mep/books/management/commands/reconcile_oclc.py

```diff
@@ -109,25 +166,44 @@ def oclc_search(self, item: Item) -> Dict:
         # search by year if known
         if item.year:
             search_opts['year'] = item.year
-
         # search year by range based on first documented event for this book
         else:
             first_date = item.first_known_interaction
             if first_date:
                 # range search ending with first known event date
                 search_opts['year'] = "-%s" % first_date.year
 
-        result = self.sru_search.search(**search_opts)
+        # filter to just periodicals if notes indicate
+        if 'PERIODICAL' in item.notes:
```

Comment:
Re: your comment @rlskoeser I think we should assume non-periodical is a book for purposes of limiting matches. It's a safe assumption and will result in fewer errant matches.